sql/moz-fx-data-shared-prod/firefox_desktop_derived/pageload_experiments_v1/query.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  *
```

Comment:
It might be useful to give the option of using a sample id
```suggestion
  * REPLACE (MOD(ABS(FARM_FINGERPRINT(document_id)), 100) AS sample_id)
```
Clustering info in metadata.yaml would need to be updated and the other two tables should have it as well for consistency

Reply:
I'd be cautions about calling this `sample_id`. If I recall correctly client_id is normally used to determine the sample_id value and I'm a bit worried that someone might use this and assume that specific sample_id in this table may correspond to the results they would get another table.

Reply:
We do use client_id to calculate sample_id but client/sample ids are null in this table so I'm not worried about trying to connect sampled results from this to another table.  On second thought though this is probably an unnecessary optimization so I think we should leave it as-is and not add complexity